### PR TITLE
add new shiny feature

### PR DESCRIPTION
**Pull request by vokilam-d**
@ljimmyl , please code review and merge!
pleaseplease!

### CODE REVIEW

File: homeWork_2/script/script.js [hunk 1]
```diff
@@ -102,8 +102,6 @@ const showSum = function(numbers = [], input = prompt('Please, enter the number'
 
         return console.log(total);
     }
-
```
Comment (ljimmyl) [inline homeWork_2/script/script.js]: remove this line

File: homeWork_2/script/script.js [hunk 2]
```diff
@@ -102,8 +102,6 @@ const showSum = function(numbers = [], input = prompt('Please, enter the number'
...
-
```
Comment (ljimmyl) [inline homeWork_2/script/script.js]: and this line too

File: homeWork_2/script/script.js (hunk 1 above)
Comment (vokilam-d) [reply homeWork_2/script/script.js]: okay okay!

File: homeWork_2/script/script.js (hunk 2 above)
Comment (vokilam-d) [reply homeWork_2/script/script.js]: sir yes sir

**Pull request merged**